src/__experimental__/components/checkbox/checkbox.style.js

```diff
@@ -15,7 +15,6 @@ const CheckboxStyle = styled.div`
   ${({
     disabled, error, warning, info, fieldHelpInline, labelSpacing, inputWidth, reverse, size, theme
   }) => css`
-    padding-top: 8px;
```

Comment:
Why was this removed? I saw you had to re-add it in configurable items. Should it not default to `pt=1`?

Reply:
The designs this story were based off had no top padding at all on checkboxes by default, will send you them on slack